server/chat-commands.js

```diff
@@ -4501,11 +4501,17 @@ const commands = {
 			target = '';
 		}
 		if (cmd === 'userdetails') {
+			if (target.length > 18) {
+				connection.send('|queryrespone|userdetails|null');
+				return false;
+			}
+
 			let targetUser = Users.get(target);
 			if (!trustable || !targetUser) {
 				connection.send('|queryresponse|userdetails|' + JSON.stringify({
 					id: target,
 					userid: toID(target),
+					name: target,
```

Comment:
Shouldn't this be the user's real name?

Reply:
Honestly I'm also kind of not particularly happy with increasing PS's overall bandwidth use for the benefit of one bot...

Reply:
Oh, this isn't the user's real name because the user's offline.